expression/builtin_math.go

```diff
@@ -966,8 +966,23 @@ func (c *randFunctionClass) getFunction(ctx sessionctx.Context, args []Expressio
 	bt := bf
 	if len(args) == 0 {
 		sig = &builtinRandSig{bt, nil}
```

Comment:
Can we init the seed here? So we don't need to clone or consider data race.

Reply:
It seems we can.